src/settings/MappingProfiles/detailsSections/view/OrderDetailsSection/EResourcesDetails.js

```diff
@@ -0,0 +1,158 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  FormattedDate,
+  FormattedMessage,
+} from 'react-intl';
+
+import {
+  Accordion,
+  Col,
+  Row,
+  KeyValue,
+} from '@folio/stripes/components';
+
+import { TRANSLATION_ID_PREFIX } from '../../constants';
+import {
+  getFieldValue,
+  getFieldValueByPath,
+  renderCheckbox,
+} from '../../utils';
+import {
+  DATE_TYPES,
+  mappingProfileFieldShape,
+} from '../../../../../utils';
+
+export const EResourcesDetails = ({ mappingDetails }) => {
+  const {
+    TWO_DIGIT,
+    NUMERIC,
+  } = DATE_TYPES;
+
+  const accessProvider = getFieldValue(mappingDetails, 'accessProvider', 'value');
+  const activated = getFieldValue(mappingDetails, 'activationStatus', 'booleanFieldAction');
+  const activationDue = getFieldValue(mappingDetails, 'activationDue', 'value');
+  const createInventory = getFieldValueByPath(mappingDetails, 'order.poLine.eresource.createInventory', 'value');
+  const materialType = getFieldValueByPath(mappingDetails, 'order.poLine.eresource.materialType', 'value');
+  const trial = getFieldValue(mappingDetails, 'trial', 'booleanFieldAction');
+  const expectedActivation = getFieldValue(mappingDetails, 'expectedActivation', 'value');
+  const userLimit = getFieldValue(mappingDetails, 'userLimit', 'value');
+  const resourceUrl = getFieldValue(mappingDetails, 'resourceUrl', 'value');
+
+  const activatedCheckbox = renderCheckbox('order.eResourcesDetails.activated', activated);
+  const trialCheckbox = renderCheckbox('order.eResourcesDetails.trial', trial);
+
+  const formattedActivationDue = (
+    <FormattedDate
+      value={activationDue}
+      day={TWO_DIGIT}
```

Comment:
Could you please investigate should we move this 3 line to 1 const and reuse it? Example below.
const BASE_FORMATTED_DATE = {
  day: TWO_DIGIT,
  month: TWO_DIGIT,
  year: NUMERIC,
};